frontend/components/ManageStakingPage/StakingContractSection/CantMigrateAlert.tsx

```diff
@@ -58,18 +60,20 @@ const AlertInsufficientMigrationFunds = ({
   if (!isAllStakingContractDetailsRecordLoaded) return null;
   if (isNil(requiredStakedOlas)) return null;
   if (isNil(safeBalance?.[TokenSymbol.OLAS])) return null;
+
   if (isNil(totalStakedOlasBalance)) return null;
 
   const requiredOlasDeposit = isServiceStaked
     ? requiredStakedOlas -
       (totalStakedOlasBalance + safeBalance[TokenSymbol.OLAS]) // when staked
     : requiredStakedOlas - safeBalance[TokenSymbol.OLAS]; // when not staked
 
-  const requiredXdaiDeposit = isInitialFunded
-    ? LOW_MASTER_SAFE_BALANCE - (safeBalance[TokenSymbol.ETH] || 0) // is already funded allow minimal maintenance
-    : (serviceFundRequirements[selectedAgentConfig.evmHomeChainId]?.[
-        TokenSymbol.ETH
-      ] || 0) - (safeBalance[TokenSymbol.ETH] || 0); // otherwise require full initial funding requirements
+  const homeChainId = selectedAgentConfig.evmHomeChainId;
+  const nativeTokenSymbol = getNativeTokenSymbol(homeChainId);
+  const requiredNativeTokenDeposit = isInitialFunded
+    ? LOW_MASTER_SAFE_BALANCE - (safeBalance[nativeTokenSymbol] || 0) // is already funded allow minimal maintenance
```

Comment:
unsure about this condition and `isInitialFunded`, can't get it logically but maybe will in the future...

Reply:
isInitialFunded is persisted on the user's machine, so they don't need to fund the monthly estimate for gas, it's only done once

it's a bit old now, and doesn't make for great UI, or even for moving forward with multi-agent imo..

but, fine for now